### PR TITLE
Initial Commit(confirmation page update):

### PR DESCRIPTION
**Pull request by kaladin9017**
#### What's this PR do?
- Add referral rewards section to confirmation page
- Add referral reward info section to confirmation page
- Add conditional to swag block partial to hide header

#### How was this tested? How should this be reviewed?
visit `/confirmation`

#### What are the relevant tickets?
https://trello.com/c/fca4080M/191-add-referrals-and-swag-info-to-the-shine-text-sign-up-confirmation-page

#### Questions / Considerations
What page should this link point to? `shinetext.com/invitefriends`

### CODE REVIEW

File: assets/styles/pages/confirmation.less [hunk 1]
```diff
@@ -190,4 +191,71 @@
...
+          width: 132px;
+
+          span {
+            font-size: 50px;
+            margin-top: -12px;
+          }
+        }
+      }
+    }
+    .referral-status-link {
```
Comment (jonuy) [inline assets/styles/pages/confirmation.less]: I think we could also add something like a `max-width: 500px` to keep the text here more compact on desktop-width screens.

File: views/confirmation.ejs [hunk 2]
```diff
@@ -2,105 +2,124 @@
...
+                  <span>
+                    0
+                  </span>
+                  / 10
+                </div>
+              </div>
+            </div>
+            <div class="referral-status-link">
+              <p>You can check your referrals progress anytime at
+                <a href="https://shinetext.com/" target="_blank">shinetext.com/invitefriends</a>
```
Comment (jonuy) [inline views/confirmation.ejs]: This should display and link to shinetext.com/referrals

File: views/confirmation.ejs [hunk 3]
```diff
@@ -2,105 +2,124 @@
...
+              <label>Text</label>
+            </li>
+          </ul>
+        </section>
+        <section>
+          <h2>Step 2. Earn swag when your friends join</h2>
+          <%- partial ('partials/swag-block.ejs') %>
+        </section>
+        <section>
+          <h2 style="padding-bottom: 20px;">Your referral count</h2>
```
Comment (jonuy) [inline views/confirmation.ejs]: To follow the design, let's capitalize the first letters on each word ("Your Referral Count")

**Comment by kaladin9017** [comment]
Updated based on review:

- Referral container is now responsive
- Capitalized the last header

File: views/confirmation.ejs [hunk 4]
```diff
@@ -2,105 +2,124 @@
...
+        <section>
+          <h2 style="padding-bottom: 20px;">Your Referral Count</h2>
+          <div class="referral-info-container">
+            <div class="next-reward">
+              <i class="reward-image-1"></i>
+
+              <div class="next-reward-count">
+                <div>
+                  <span>
+                    0
```
[the remark this answers is not in the document]
Comment (jonuy) [reply views/confirmation.ejs]: Good catch. In the API calls that are made during the sign up process, are we getting a user's current referral count there? This should be an accurate representation of what a user currently has. There are some cases where existing users could be going back through the sign up flow, and we should accurately show their count here.

If not, we'll want to run by Mia first, but I think we could just remove the "0/10" portion of this and update copy with what Sandy recommended above.

[the remark this answers is not in the document]
Comment (kaladin9017) [reply views/confirmation.ejs]: Makes sense. Updated the first header & the copy for the last block. Checked with mia and we are good to go.

**Pull request merged**